apache/superset
# feat: scroll to bottom when adding a new native filter and the page is filled

### SUMMARY

When adding a native filter, and the page is already filled (with other filters), auto-scroll to the bottom so that the newly added filter is visible on the left panel.

Also, fixes a typo in the onRearra**n**ge function.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:

https://user-images.githubusercontent.com/17252075/157144489-9c4b5eef-f1b9-41cb-bca5-d9a7c9a5bac1.mov

After:

https://user-images.githubusercontent.com/17252075/157144137-e2a3e265-97eb-40fb-a18e-23a8d6386951.mov

### TESTING INSTRUCTIONS

 1. Go to any dashboard
 2. Hit the Add/Edit filters button on the left panel
 3. Start adding filters & dividers
 4. Ensure that, when the filter container fills the screen, adding new filters scrolls the container to the bottom

### ADDITIONAL INFORMATION
